javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,146 @@
+/*freecodecamp.org account name: mehmetnuricavdar
+https://www.freecodecamp.org/mehmetnuricavdar
+*/
+
+// ######    Flight booking fullname function
+
+function getFullname(firstname, surname) {
+  const fullName = `${firstname} ${surname}`;
+  return fullName;
```

Comment:
You can return it in one line instead of creating a variable and then returning that variable,
like this
`return '${firstname} ${surname}'`

Reply:
i guess i tried it it didn't work but now it worked. Possibly i did a typo.